### PR TITLE
single container: add grafana

### PR DESCRIPTION
**Pull request by uwedeportivo**
Add grafana to the single container deploy context.

### CODE REVIEW

File: cmd/server/build.sh [hunk 1]
```diff
@@ -60,6 +60,9 @@ echo "--- prometheus config"
 mkdir "$OUTPUT/etc"
 cp -r dev/prometheus "$OUTPUT/etc"
 
+echo "--- grafana config"
+cp -r dev/grafana "$OUTPUT/etc"
```
Comment (keegancsmith) [inline cmd/server/build.sh]: looks like you may have forgotten to include these files in your PR

**Comment by uwedeportivo** [comment]
to try it out:

`
docker run --publish 7080:7080 --publish 2633:2633  --publish 3000:3000 --publish 9090:9090 --rm --volume ~/.sourcegraph/config:/etc/sourcegraph --volume ~/.sourcegraph/data:/var/opt/sourcegraph --volume ~/sourcegraph-docker/prometheus-disk:/prometheus-data --volume  ~/sourcegraph-docker/grafana-disk:/var/lib/grafana sourcegraph/server:3.6.2-internal-grafana
`

File: cmd/server/build.sh (hunk 1 above)
Comment (uwedeportivo) [reply cmd/server/build.sh]: you mean dev/grafana ? that one already exists (i'm reusing the same as the dev launch deploy context)

File: cmd/server/shared/shared.go [hunk 2]
```diff
@@ -151,6 +151,7 @@ func Main() {
 		`repo-updater: repo-updater`,
 		`syntect_server: sh -c 'env QUIET=true ROCKET_ENV=production ROCKET_PORT=9238 ROCKET_LIMITS='"'"'{json=10485760}'"'"' ROCKET_SECRET_KEY='"'"'SeerutKeyIsI7releuantAndknvsuZPluaseIgnorYA='"'"' ROCKET_KEEP_ALIVE=0 ROCKET_ADDRESS='"'"'"127.0.0.1"'"'"' syntect_server | grep -v "Rocket has launched" | grep -v "Warning: environment is"' | grep -v 'Configured for production'`,
 		`prometheus: prometheus -config.file=/etc/prometheus/prometheus_local.yml  -storage.local.path=/prometheus-data -web.console.libraries=/etc/prometheus/console_libraries  -web.console.templates=/etc/prometheus/consoles`,
+		`grafana: env GF_AUTH_ANONYMOUS_ENABLED=true GF_AUTH_ANONYMOUS_ORG_NAME=Sourcegraph GF_AUTH_ANONYMOUS_ORG_ROLE=Editor GF_USERS_ALLOW_SIGN_UP=false GF_USERS_AUTO_ASSIGN_ORG=true GF_USERS_AUTO_ASSIGN_ORG_ROLE=Editor /usr/share/grafana/bin/grafana-server -config /etc/grafana/grafana.ini -homepath /usr/share/grafana`,
```
Comment (tsenart) [inline cmd/server/shared/shared.go]: Where does `/etc/grafana/grafana.ini` come from?

Comment (uwedeportivo) [reply cmd/server/shared/shared.go]: dev/grafana/grafana.ini

**Comment by uwedeportivo** [comment]
vielen dank for the review :-)

**Pull request merged**